src/main/java/pl/noip/lolstats/lol/stats/service/RiotDataClient.java

```diff
@@ -0,0 +1,31 @@
+package pl.noip.lolstats.lol.stats.service;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.*;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+import pl.noip.lolstats.lol.stats.dto.SummonerDataResponse;
+
+@Component
+@Slf4j
+public class RiotDataClient {
```

Comment:
Merge RiotDataClient and RiotMatchesClient to one class called e.g. RiotRestClient witch various methods for different kinds of data.